classes/Player.py

```diff
@@ -32,14 +35,35 @@ def bullet_shoot(self,bullet_group):
            Bullet(self.rect.midtop,bullet_group)
            self.bullet_sound.play()
 
-    def hit_target(self,target_group):
+    def hit_target(self,target_group,display_surface):
         if pygame.sprite.spritecollide(self,target_group,False):
-            pygame.quit()
-            sys.exit()
+            self.game_over=True
+            game_over_text = f'GameOver!!'
```

Comment:
I guess it is missing a space between words- Game Over would be more appropriate